src/main/resources/openshift/deploy.sh

```diff
@@ -2,7 +2,9 @@
 
 set -e
 
-OCP_PROJECT=rhamt
+if [ -z "$OCP_PROJECT" ]; then
```

Comment:
log contains 
`10:55:27,993 INFO [org.wildfly.extension.undertow] (ServerService Thread Pool -- 64) WFLYUT0014: Creating file handler for path '/opt/eap/windup-web-redirect' with options [directory-listing: 'false', follow-symlink: 'false', case-sensitive: 'true', safe-symlink-paths: '[]']`

so not sure what happened that the standalone-full.xml has not the redirect in it.